packages/meditrak-app/app/sync/ChangeQueue.js

```diff
@@ -52,12 +52,12 @@ export class ChangeQueue {
     const allChanges = this.database.objects('Change').sorted('timestamp');
     const changesWithinThreshold = [];
     let size = 0;
-    for (let changeId in allChanges) {
+    for (const changeId in allChanges) {
       const change = allChanges[changeId];
```

Comment:
```suggestion
    for (const change of allChanges) {
```
This is a substantive rather than formatting change, but I think it is appropriate

Reply:
Good point! Will do